pages/draft-06/json-hyper-schema-release-notes.md

```diff
@@ -1,107 +1,104 @@
 
-#### Changes from draft-05 to draft-06
+##### Changes from draft-05 to draft-06
 
-keyword | change | consequence
----- | ---- | ----
-`"method"` | *removed* | for HTTP method proposals, see issues [#73](https://github.com/json-schema-org/json-schema-spec/issues/73) and [#296](https://github.com/json-schema-org/json-schema-spec/issues/296) (use either `"method"` or `"allow"` as an extension keyword if needed); indication of how to use `"schema"` and `"encType"` no longer necessary
-`"schema"` | *removed* | use `"hrefSchema"`, `"submissionSchema"`,  or `"targetSchema"` |
-`"encType"` | *removed* | use `"submissionEncType"` for request bodies; no longer needed for URI query strings
-`"hrefSchema"` | *added* | replaces `"method": "get", "schema": {...}`, with additional functionality |
-`"submissionSchema"` | *added* | replaces `"method": "post", "schema": {...}`
-`"submissionEncType"` | *added* | replaces `"method": "post", "encType": "..."`
-`"href"` | preprocessing removed | *to be replaced and expanded in future drafts*
+| keyword               | change                | consequence                                                                                                                                                                                                                                                                                                                           |
+| --------------------- | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `"method"`            | _removed_             | for HTTP method proposals, see issues [#73](https://github.com/json-schema-org/json-schema-spec/issues/73) and [#296](https://github.com/json-schema-org/json-schema-spec/issues/296) (use either `"method"` or `"allow"` as an extension keyword if needed); indication of how to use `"schema"` and `"encType"` no longer necessary |
+| `"schema"`            | _removed_             | use `"hrefSchema"`, `"submissionSchema"`, or `"targetSchema"`                                                                                                                                                                                                                                                                         |
+| `"encType"`           | _removed_             | use `"submissionEncType"` for request bodies; no longer needed for URI query strings                                                                                                                                                                                                                                                  |
```

Comment:
question: why are the changes italicized?

Reply:
I can't answer this. That was the style guide adopted by the previous draft writer.